homeassistant/components/vesync/sensor.py

```diff
@@ -49,14 +47,13 @@
 class VeSyncSensorEntityDescription(SensorEntityDescription):
     """Describe VeSync sensor entity."""
 
-    value_fn: Callable[[VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch], StateType]
+    value_fn: Callable[
+        [VeSyncBaseDevice],
+        StateType,
+    ]
```

Comment:
```suggestion
    value_fn: Callable[[VeSyncBaseDevice], StateType]
```

Reply:
This is due to automatic formatting in Code. I think it is coming from Ruff extension which is enabled on `Dev Container: Home Assistant Dev`. The ruff-format commit check also doesn't complain about this. Although I am not sure if it is supposed to.